### PR TITLE
Add script for printing example.test.txt

### PR DESCRIPTION
**Pull request by raidgar98**
## What has changed?

Add new script that prints content of example.test.txt or any text file

## How it was achieved?

By adding new `printer.py` script which can be run via `python printer.py`

## How to test it?

Create example file, and check is output from program same

### CODE REVIEW

**Review by SKNGetIT — DISMISSED** [review]

File: printer.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+with open('example.test.txt', 'r') as file:
```
Comment (raidgar98) [inline printer.py]: In what mode did you open this file? text or binary

```
with open('example.test.txt', 'rt') as file:  
```

File: printer.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+with open('example.test.txt', 'r') as file:    
+	for line in file:    
+		print(line)
```
Comment (raidgar98) [inline printer.py]: please use logging library

File: printer.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
-with open('example.test.txt', 'r') as file:
```
Comment (raidgar98) [inline printer.py]: Please squash with previous commit

File: printer.py [hunk 4]
```diff
@@ -1,4 +1,8 @@
-file_name = 'example.test.txt'
+from sys import argv
```
Comment (raidgar98) [inline printer.py:1]: please, don't use sys.argv, use dedicated library: [argparse](https://docs.python.org/3/library/argparse.html)

File: printer.py [hunk 5]
```diff
@@ -5,4 +5,4 @@
 
 with open(file_name, 'r') as file:
 	for line in file:
-		print(line)
+		print(len(line), line)
```
Comment (raidgar98) [inline printer.py:10]: please update this string to be more readable, like:
f'lenght={len(line)}, string={line}'

File: printer.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+from sys import argv
+
+print("This is version 1.0")
+
+DEFAULT_FILE_NAME = 'example.test.txt'
+file_name = argv[1] if len(argv) > 1 else DEFAULT_FILE_NAME
+
+with open(file_name, 'r') as file:
+	for line in file:
+		print(len(line), line)
```
Comment (raidgar98) [inline printer.py:10]: please strip newline from line

**Review by raidgar98 — COMMENTED** [review]
Please, remember to squash commit: 0ae7b73f3d27345da81b0982d030ac94d707f01b

**Pull request merged**